### PR TITLE
[4.0] Make Batch button to respect "listCheck" option

### PR DESCRIPTION
**Pull request by Fedik**
Pull Request for Issue #21109 .

### Summary of Changes
Now the Batch button is grayed without selected Article

### CODE REVIEW

File: layouts/joomla/toolbar/popup.php [hunk 1]
```diff
@@ -34,22 +34,19 @@
...
 {
-	Text::script('JLIB_HTML_PLEASE_MAKE_A_SELECTION_FROM_THE_LIST');
-	Text::script('ERROR');
-	$message = "{'error': [Joomla.JText._('JLIB_HTML_PLEASE_MAKE_A_SELECTION_FROM_THE_LIST')]}";
-	$alert = 'Joomla.renderMessages(' . $message . ')';
-
-	$modalAttrs['onclick'] = <<<JS
-if (document.adminForm.boxchecked.value==0){ $alert } else { jQuery( '#$selector' ).modal('show'); return true; }
-JS;
+	$modalAttrs['onclick'] = 'jQuery(\'#' . $selector . '\' ).modal(\'show\');';
```
Comment (dgrammatiko) [inline layouts/joomla/toolbar/popup.php]: How about `document.getElementById("' . $selector . '").open();`

Comment (Fedik) [reply layouts/joomla/toolbar/popup.php]: I have ignored this part, I am already got lost in all the modal changes :smile: 
Thanks for note, I will change it.

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 0b4608c447278dfc4df1425ba4eccc005c61a78c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21284">issues.joomla.org/tracker/joomla-cms/21284</a>.</sub>

**Comment by Fedik** [comment]
conflict fixed, 
note about testing:
after apply the patch you have to run `npm install`, to see a result

**Comment by ghost** [comment]
@Fedik thanks for Info " after apply the patch you have to run npm install, to see a result".

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on c4bb28aa1256364e58d36645441049e01736d9fe<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21284">issues.joomla.org/tracker/joomla-cms/21284</a>.</sub>

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on c4bb28aa1256364e58d36645441049e01736d9fe<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21284">issues.joomla.org/tracker/joomla-cms/21284</a>.</sub>

**Comment by ghost** [comment]
Ready to Commit after two successful tests.

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**